com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -289,7 +294,7 @@ private void OnValidate()
                     {
                         {
                             var childNetworkObjects = new List<NetworkObject>();
-                            networkPrefab.Prefab.GetComponentsInChildren(/* includeInactive = */ true, childNetworkObjects);
+                            networkPrefab.Prefab.GetComponentsInChildren( /* includeInactive = */ true, childNetworkObjects);
```

Comment:
very minor pet peeve: those whitespace changes can be removed before committing/pushing. It's really not an issue often, but sometimes it can be the unlucky line that conflicts with another change and causes merge issues down the road.

Reply:
always rely on `./standard.py --fix` to have consistent whitespaces! :)